base_tier_validation/models/tier_review.py

```diff
@@ -141,7 +135,10 @@ def _compute_todo_by(self):
         for rec in self:
             todo_by = False
             if rec.reviewer_group_id:
-                todo_by = _("Group %s") % rec.reviewer_group_id.name
+                todo_by = self.env._(
+                    "Group %(reviewer_group_name)s",
+                    reviewer_group_name=rec.reviewer_group_id.name,
```

Comment:
I'd keep the original translatable term here, to not invalidate existing translations.
There's really no benefit in using kword arguments when there's only one param

```py
todo_by = this.env._("Group %s", rec.reviewer_group_id.name)
```